client/src/utils/api.js

```diff
@@ -174,3 +180,54 @@ export const deleteResource = (id: string): ApiResponse<null> => {
       },
     );
 };
+
+export const getTranslationByLanguage = (
+  language: string,
+): ApiResponse<Translation> => {
+  const requestExtension = `/api/translation/?language=${language}`;
```

Comment:
I think we can remove the extra `/` here before the question mark just to keep with better HTTP standards - so it's `/api/translation?language=${language}` (this should still work, I tested on Postman)